Aspire Budgeting/Models/AccountBalancesMetadata.swift

```diff
@@ -22,7 +22,9 @@ struct AccountBalancesMetadata {
   private static func parse(metadata: [[String]]) -> [AccountBalance] {
     var accountBalances = [AccountBalance]()
     for row in metadata {
-      accountBalances.append(AccountBalance(accountName: row[0], balance: row[1]))
+      if row.count == 2 {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/67edf3325570da301c03450fdbbe6d8b96d1b452/Aspire Budgeting/Models/AccountBalancesMetadata.swift#L25"></span><code>where</code> clauses are preferred over a single <code>if</code> inside a <code>for</code>.<br />
<code>for_where</code> <code>AccountBalancesMetadata.swift:25</code></td>
    </tr>
  </tbody>
</table>